Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -174,6 +174,16 @@ void FSpatialNetGUIDCache::RemoveEntityNetGUID(Worker_EntityId EntityId)
 	UnrealObjectRef* ActorRef = NetGUIDToUnrealObjectRef.Find(EntityNetGUID);
 	NetGUIDToUnrealObjectRef.Remove(EntityNetGUID);
 	UnrealObjectRefToNetGUID.Remove(*ActorRef);
+
+	UnrealObjectRef SubobjectRef(ActorRef->Entity, ActorRef->Offset + 1);
```

Comment:
ActorRef->Offset should always be zero here, I think we should check that, and just pass 1 in directly to the subobject offset.